README.md

```diff
@@ -66,6 +64,15 @@ This plugin uses the AWS SDK Go, and uses its [default credential provider chain
 * `FLB_LOG_LEVEL`: Set the log level for the plugin. Valid values are: `debug`, `info`, and `error` (case insensitive). Default is `info`. **Note**: Setting log level in the Fluent Bit Configuration file using the Service key will not affect the plugin log level (because the plugin is external).
 * `SEND_FAILURE_TIMEOUT`: Allows you to configure a timeout if the plugin can not send logs to CloudWatch. The timeout is specified as a [Golang duration](https://golang.org/pkg/time/#ParseDuration), for example: `5m30s`. If the plugin has failed to make any progress for the given period of time, then it will exit and kill Fluent Bit. This is useful in scenarios where you want your logging solution to fail fast if it has been misconfigured (i.e. network or credentials have not been set up to allow it to send to CloudWatch).
 
+### Templating Log Group and Stream Names
```

Comment:
I want this section to have a full example with an example config (after the explanation). 

Show an example where they use both a key from the record and ECS Metadata in the names. Show the output configuration, an example log record, and what the resulting stream/group name will be.